### DISCUSSION TITLE
Can not import Fraction package when Parcel build but working in Parcel start

### DISCUSSION THREAD

**Discussion by liyongxing1410** [parcel-bundler/parcel#6985] [Q&A]
I have a problem when importing Fraction.
I used Parcel to bundle my static web project and it works fine when running using Parcel dev: parcel index.html
But when build by Parcel: parcel build index.html
It generates a dist folder when i deployed it to netlify.app. It give me an error in console: Fraction is not defined.
It seem that there is a problem in importing Fraction and could you help me out with this? Thanks.

![Ảnh chụp Màn hình 2021-09-25 lúc 17 10 09](https://user-images.githubusercontent.com/89700043/134768186-ed4a39f2-485b-493a-a287-3e2e14eba747.png)
![Ảnh chụp Màn hình 2021-09-25 lúc 17 10 30](https://user-images.githubusercontent.com/89700043/134768190-a9f504cb-da40-4656-9f47-2056794787c2.png)

**Comment by mischnic** [comment]
Have you seen https://github.com/parcel-bundler/parcel/issues/6851#issuecomment-910591126?
